package.json

```diff
@@ -51,6 +51,7 @@
         "@commitlint/cli": "^8.3.5",
         "@commitlint/config-conventional": "^8.2.0",
         "@types/classnames": "^2.2.10",
+        "@types/draft-js": "^0.10.40",
```

Comment:
Usually want to keep types at the same version as their associated dependency. It looks like we should also hardcore our version to `"draft-js": "@0.10.5"`, since that's what our parent app is using.

Reply:
> Usually want to keep types at the same version as their associated dependency. It looks like we should also hardcore our version to `"draft-js": "@0.10.5"`, since that's what our parent app is using.

The version of `@types/draft-js` and `draft-js` don't match actually. draft-js 0.10.5 is published 2 years ago and @types is published 3 years ago. @types 0.10.40 is for ts@2.9 - 3.9. And we're using ts@3.8. So I think we should keep using @types 0.10.40